aws-lc-rs/src/cipher/streaming.rs

```diff
@@ -631,6 +668,61 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_step_cfb128() {
+        let random = SystemRandom::new();
+        let mut key = [0u8; AES_256_KEY_LEN];
+        random.fill(&mut key).unwrap();
+
+        let encrypting_key_creator = || {
+            let key = UnboundCipherKey::new(&AES_256, &key.clone()).unwrap();
+            StreamingEncryptingKey::cfb128(key).unwrap()
+        };
+        let decrypting_key_creator = |decryption_ctx: DecryptionContext| {
+            let key = UnboundCipherKey::new(&AES_256, &key.clone()).unwrap();
+            StreamingDecryptingKey::cfb128(key, decryption_ctx).unwrap()
+        };
+
+        for i in 13..=21 {
+            for j in 124..=131 {
```

Comment:
Sorry if I've asked this before and I've noticed this is used in the other tests as well. 
What do these pair of numbers mean and should we abstract them into variables?